.gitignore

```diff
@@ -37,6 +37,10 @@ CMakeFiles
 # files commonly related to building out-of-tree
 /build/
 
+# files produced by lit testing
+**/Output
```

Comment:
We already a have `out/test` for test output.  Can we teach lit to use the same directory?

Reply:
Possibly. I will investigate.

Reply:
It turns out that this "Output" directory is hard-coded into lit.

Reply:
Oh.. that is odd.. and uppercase output non-the-less.   I guess I'd never realized that "Output" was part of all my tests intermediates in llvm.

In any case this `Output` directory looks like lives under the build directory so this is only needs for in-tree builds so it can go under the `files related to building in-tree` category. 

It would be good if we could make the current check.py tests also write to the build director and use this same sub-directory name (maybe as a precursor to this change).